lib/modules/manager/kustomize/extract.ts

```diff
@@ -144,11 +145,25 @@ export function extractHelmChart(
     return null;
   }
 
+  if (isOCIRegistry(helmChart.repo)) {
+    return {
+      depName: helmChart.name,
+      currentValue: helmChart.version,
+      datasource: DockerDatasource.id,
+      packageName: `${helmChart.repo.replace('oci://', '')}/${helmChart.name}`,
+      // https://github.com/helm/helm/issues/10312
+      // https://github.com/helm/helm/issues/10678
+      pinDigests: false,
+      depType: 'OCIChart',
```

Comment:
🤔 isn't that a normal helm chart? I don't see a benefit to use a different type here.
```suggestion
      depType: 'HelmChart',
```

Reply:
I was using it in my `packageRules` but I realized it's not entirely necessary for my use-case anymore.